packages/owl-bot/src/cmd.ts

```diff
@@ -19,12 +19,20 @@ export type Cmd = (
   options?: proc.ExecSyncOptions | undefined
 ) => Buffer;
 
+function sanitize(command: string) {
+  command = command.replace(
+    /x-access-token:([^/]+)/,
```

Comment:
I noticed the token shows up in logs, not a huge deal in cloud build since it's short lived ... but probabluy good to redact it.